crowbar_framework/app/controllers/api/upgrade_controller.rb

```diff
@@ -19,6 +19,29 @@ class Api::UpgradeController < ApiController
 
   api :GET, "/api/upgrade", "Show the Upgrade status object"
```

Comment:
So `/api/upgrade` also contains `/api/crowbar/upgrade` results? Could you add that as a comment in the example?